template/src/ontology/run.sh.jinja2

```diff
@@ -16,6 +16,14 @@
 
 set -e
 
+# Check for spaces in the current directory path
+for item in $PWD ; do
+    if [ "$item" != "$PWD" ]; then
```

Comment:
I don't get how this fixes the issue, but I trust that it does and I'm just misunderstanding something. Thanks for addressing #1078!

Reply:
This is a way of checking whether the current directory (as stored in the `PWD` variable) contains spaces.

To illustrate how it works, let’s first suppose that `PWD` does _not_ contain any space (for example, let’s say it contains `/home/alice/myont`). The first line will be expanded as `for item in /home/alice/myont`. Because `/home/alice/myont` cannot be split by the shell (since it contains no spaces), the `for` loop will be executed only once, where the loop variable `item` will be set to `/home/alice/myont`.

Inside the loop, the `"$item" != "$PWD"` condition will be evaluated to _false_ (since `item` contains the same value as `PWD`), so we’ll leave the loop without having done anything.

But suppose that `PWD` contains (at least) one space (for example `/home/alice/my ont`). The first line will be expanded as `for item in /home/alice/my ont`. The shell will split `/home/alice/my ont` in two (notice that `$PWD` is _not_ enclosed in quotes), so the `for` loop will (in principle) be executed twice: first with `item` set to `/home/alice/my`, and then with `item` set to `ont`.

During the first turn in the loop, the `"$item" != "$PWD"` condition will now be evaluated to _true_, so the script will print the error message and terminate.

That’s a slightly convoluted way of doing `if echo $PWD | grep ' ' ; then echo "There are spaces in the current directory path ; fi`, with the advantage that we don’t need to spawn a `grep` process. :)

Reply:
Wow, thanks for taking the time on the explanation!
I only needed to read up to:
>  The first line will be expanded as `for item in /home/alice/myont`.

I though that `$PWD` would get interpolated as a string, so I thought it would be iterating over characters int hat string.